dashboard/test/models/user_level_info_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserLevelInfoTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
Do we keep these Rails auto generated files around even if we don't have any content in them?

Reply:
🤷‍♀ I was assuming that as I added things to the table I would probably need to use them for testing?

Reply:
Ah, I didn't realize these were auto generated! Perhaps your first test can just make sure that we can create a user_level_info using the new factory method?